README.md

```diff
@@ -16,7 +16,7 @@ Documentation is available at [HexDocs](https://hexdocs.pm/membrane_h264_ffmpeg_
 Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
 
 ```elixir
-  {:membrane_h264_ffmpeg_plugin, "~> 0.22.1"}
+  {:membrane_h264_ffmpeg_plugin, "~> 0.23.1"}
```

Comment:
If it's supposed to be 0.23.0, then
```suggestion
  {:membrane_h264_ffmpeg_plugin, "~> 0.23.0"}
```

Reply:
I rebased on master, there is 0.23.0 version already there.